ci/ci/ci.py

```diff
@@ -767,6 +764,9 @@ async def on_startup(app):
         )
     ]
 
+    app['task_manager'] = aiotools.BackgroundTaskManager()
```

Comment:
gotta keep task manager up above b/c of the update envoy configs and cleanup expired namespaces.

Reply:
right right